PSR2R/Sniffs/Classes/InterfaceNameSniff.php

```diff
@@ -11,18 +12,12 @@ class InterfaceNameSniff extends AbstractSniff {
 
 	/**
 	 * @inheritDoc
+	 * @return void
```

Comment:
I did not think so, but one of the PSR2R sniffs complains without it. The unit tests do not show that error, so it must be one of the other rules mentioned in ruleset.xml. So far I've not figured out which rule is the problem.